src/lib/features/snackbar/index.ts

```diff
@@ -0,0 +1,3 @@
+export type { SnackbarOptions } from './model/types';
```

Comment:
[about this whole slice] 

issue: I don't think `snackbar` is a feature of our application. Snackbars by itself don't really add any business logic to us.

suggestion: move this slice into `shared/ui`

Reply:
But why to `shared/ui`? If would feel strange to have `shared/ui/snackbar/model/store.ts` file

Reply:
True, you can move the store outside of the UI segment and into the model in shared. It is just not a feature, in my opinion